DMCompiler/DM/Visitors/DMVisitorExpression.cs

```diff
@@ -360,8 +360,8 @@ class DMVisitorExpression : DMASTVisitor {
 
         public void VisitTernary(DMASTTernary ternary) {
             var a = DMExpression.Create(_dmObject, _proc, ternary.A, _inferredPath);
-            var b = DMExpression.Create(_dmObject, _proc, ternary.B, _inferredPath);
-            var c = DMExpression.Create(_dmObject, _proc, ternary.C, _inferredPath);
+            var b = DMExpression.Create(_dmObject, _proc, ternary.B ?? new DMASTConstantNull(ternary.Location), _inferredPath);
```

Comment:
A missing `c` expression is valid, a missing `b` expression is not.
```suggestion
            var b = DMExpression.Create(_dmObject, _proc, ternary.B ?? new DMASTConstantNull(ternary.Location), _inferredPath);
```
```suggestion
            var b = DMExpression.Create(_dmObject, _proc, ternary.B, _inferredPath);
```